compendium/DeclarativeServices/src/manager/states/CMEnabledState.cpp

```diff
@@ -98,7 +103,11 @@ void CMEnabledState::DeleteConfigurations()
   auto fut = GetFuture();
   if(fut.valid())
   {
-    fut.get(); // wait for the configurations to become available
+    try {
+      fut.get(); // wait for the configurations to become available
+    } catch (...) {
+      std::cout << "exception here" << std::endl;
```

Comment:
is this leftovers from debugging?

in general, don't write to `std::cout`, log it using the LogService

Reply:
yup, forgot to remove from debugging. removed the unnecessary try/catch